cmd/security-secrets-setup/Dockerfile

```diff
@@ -50,6 +51,10 @@ LABEL license='SPDX-License-Identifier: Apache-2.0' \
 
 USER root
 
+# install necessary tools
+RUN apk update && apk add ca-certificates dumb-init jq \
```

Comment:
It is more efficient code to write
``
apk add --no-cache ca-certificates dumb-init jq
```

Instead of writing apk update before and rm /var/cache/apk after.

Source: https://stackoverflow.com/questions/49118579/alpine-dockerfile-advantages-of-no-cache-vs-rm-var-cache-apk

Reply:
I use `rm` literally since `--no-cache` is not working for me as expected locally here.  When I used `--no-cache` and then tried to run docker image exec, the cache folder still there not being cleaned up or deleted for some reason. Maybe you can try and see.